src/components/tasks/TaskSearchDev/useFilterSuggestion.tsx

```diff
@@ -64,12 +66,41 @@ export const useFilterSuggestion = ({
                 key = potentialKey.trim();
                 userInput = values.join(':').trimStart();
             }
+
+            return {
+                key,
+                value: userInput,
+            };
         }
-        if (userInput.length > 2) {
+
+        return {
+            key: '',
+            value: userInput,
+        };
+    }
+
+    function getAssigneeSuggestions() {
+        const result =
+            searchedUsers?.users
+                ?.filter(
+                    (user) =>
+                        !selectedFilters.some(
+                            (selected) => selected.assignee === user.username
+                        )
+                )
+                .map((user) => ({
+                    assignee: user.username || user.github_display_name,
+                })) || [];
+
+        return result;
+    }
```

Comment:
please add meaningful variable names, its not the .filter and .map is doing here

Reply:
Please check now